Apps/Src/Voltage.c

```diff
@@ -61,7 +62,12 @@ void Voltage_Init(cell_asic *boards){
 	// release mutex
   	OSMutexPost(&MinionsASIC_Mutex, OS_OPT_POST_NONE, &err);
   	assertOSError(err);
-	
+
+	for (int i = 0; i < NUM_BATTERY_MODULES_PER_MINION; i++){
+		open_wire_mask <<= 1;
+		open_wire_mask |= 1;
+	}
```

Comment:
```suggestion
	// set the lower NUM_BATTERY_MODULES_PER_MINION bits of open_wire_mask to 1
	open_wire_mask = 1;
	open_wire_mask <<= NUM_BATTERY_MODULES_PER_MINION + 1;
	open_wire_mask -= 1;
```
I think this will accomplish the same thing more efficiently (except in the edge cases where `NUM_BATTERY_MODULES_PER_MINION > 31`, which is not supported by our hardware anyway). Not sure if this is worth it if it reduces the code readability too much, since it is only called once.

Reply:
After looking at this some more, the open wire mask is only being used for the last minion board, so we should be generating this based on `NUM_BATTERY_MODULES % NUM_BATTERY_MODULES_PER_MINION` instead of `NUM_BATTERY_MODULES_PER_MINION`. For example:

```suggestion
	// set the lower bits of open_wire_mask to 1, such that there are as many 1s are modules connected to the last minion
	open_wire_mask = 1;
	open_wire_mask <<= (NUM_BATTERY_MODULES % NUM_BATTERY_MODULES_PER_MINION) + 1;
	open_wire_mask -= 1;